pipeline-braille-scripts/dtbook-to-pef/src/main/resources/xml/xproc/dtbook-to-pef.xpl

```diff
@@ -29,537 +29,90 @@
-        <p:documentation xmlns="http://www.w3.org/1999/xhtml">
-            <h2 px:role="name">Output directory</h2>
-            <p px:role="desc">Directory for storing result files.</p>
-        </p:documentation>
-    </p:option>
-    <p:option name="temp-dir" required="false" px:output="temp" px:type="anyDirURI" select="''">
-        <p:documentation xmlns="http://www.w3.org/1999/xhtml">
-            <h2 px:role="name">Temporary directory</h2>
-            <p px:role="desc">Directory for storing temporary files.</p>
-        </p:documentation>
-    </p:option>
+    <p:option name="stylesheet"/>
+    <p:option name="transform"/>
+    <p:option name="ascii-table"/>
+    <p:option name="include-preview"/>
+    <p:option name="include-brf"/>
+    <p:option name="page-width"/>
+    <p:option name="page-height"/>
+    <p:option name="predefined-page-formats"/>
+    <p:option name="left-margin"/>
+    <p:option name="duplex"/>
+    <p:option name="levels-in-footer"/>
+    <p:option name="main-document-language"/>
+    <p:option name="hyphenation"/>
+    <p:option name="line-spacing"/>
+    <p:option name="tab-width"/>
+    <p:option name="capital-letters"/>
+    <p:option name="accented-letters"/>
+    <p:option name="polite-forms"/>
+    <p:option name="downshift-ordinal-numbers"/>
+    <p:option name="include-captions"/>
+    <p:option name="include-images"/>
+    <p:option name="include-image-groups"/>
+    <p:option name="include-line-groups"/>
+    <p:option name="text-level-formatting"/>
+    <p:option name="include-note-references"/>
+    <p:option name="include-production-notes"/>
+    <p:option name="show-braille-page-numbers"/>
+    <p:option name="show-print-page-numbers"/>
+    <p:option name="force-braille-page-break"/>
+    <p:option name="toc-depth"/>
+    <p:option name="ignore-document-title"/>
+    <p:option name="include-symbols-list"/>
+    <p:option name="choice-of-colophon"/>
+    <p:option name="footnotes-placement"/>
+    <p:option name="colophon-metadata-placement"/>
+    <p:option name="rear-cover-placement"/>
+    <p:option name="number-of-pages"/>
+    <p:option name="maximum-number-of-pages"/>
+    <p:option name="minimum-number-of-pages"/>
+    <p:option name="sbsform-macros"/>
+    <p:option name="output-dir"/>
+    <p:option name="brf-output-dir" required="false"/>
+    <p:option name="html-output-dir" required="false"/>
+    <p:option name="temp-dir" required="false"/>
     
     <!-- ======= -->
     <!-- Imports -->
     <!-- ======= -->
-    <p:import href="dtbook-to-pef.convert.xpl"/>
+    <p:import href="http://www.daisy.org/pipeline/modules/braille/dtbook-to-pef/library.xpl">
+        <!-- NOTE: we cannot use a relative url to import dtbook-to-pef.convert.xpl directly here
+                   because this script uses px:extends-script in the XML catalog which
+                   changes the base URI of the script at build time. -->
```

Comment:
Yeah I was thinking if that could be possible but I don't think there's a 100% good fix. What would improve the situation though would be to go through all `p:document` and `p:import` elements and re-format any relative URIs. The issue is that the XProc script that is generated is "moved" to `/generated-scripts/`, which means any relative URIs no longer work.

One alternative could maybe be to overwrite the original script instead of moving it to `/generated-scripts/`.